spec/services/auctions_closed_within_last_twenty_four_hours_finder_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe AuctionsClosedWithinLastTwentyFourHoursFinder do
+  describe '#perform' do
+    it 'returns auctions with an end datetime that occured today' do
+      Timecop.freeze(Time.parse("19:00:00 UTC")) do
```

Comment:
One idea for improvement: whatever the time is that we send it should be a constant that we reference here